scripts/server_scripts/mock_health_event_publisher.py

```diff
@@ -0,0 +1,248 @@
+# cortx-questions@seagate.com.
+
+
+"""Module helps in generating the mock health event and publishing it to the message bus."""
+# This script is provided by HA. If any changes are done by HA Dev team, script needs to be
+# copied from cortx-ha/ha/util/health_generator/mock_health_event_publisher.py
+
+
+import argparse
+import sys
+import pathlib
+import errno
+import json
+import time
+
+from cortx.utils.conf_store import Conf
+from cortx.utils.event_framework.health import HealthAttr, HealthEvent
+from ha.util.conf_store import ConftStoreSearch
+from ha.core.config.config_manager import ConfigManager
+from ha import const
+from ha.util.message_bus import MessageBus
+from ha.core.system_health.const import HEALTH_STATUSES
+
+DOCKER_ENV_FILE = '/.dockerenv'
+_INDEX = 'cortx'
+CLUSTER_CONFIG_FILE = '/etc/cortx/cluster.conf'
+CONFIG_FILE_FORMAT = 'yaml'
+_EVENTS_KEY = 'events'
+_SOURCE_KEY = 'source'
+_NODE_ID_KEY = 'node_id'
+_RESOURCE_TYPE_KEY = 'resource_type'
+_RESOURCE_ID_KEY = 'resource_id'
+_RESOURCE_STATUS_KEY = 'resource_status'
+_SPECIFIC_INFO_KEY = 'specific_info'
+_DELAY_KEY = 'delay'
+
+
+def is_container_env() -> bool:
+    """Returns True if environment is docker container else False."""
+    docker_env_file_path = pathlib.Path(DOCKER_ENV_FILE)
+    if docker_env_file_path.exists():
+        return True
+    return False
+
+
+# pylint: disable=redefined-outer-name
+def get_data_nodes() -> list:
+    """Fetches data node ids using HA wrapper class and displays the result.
+
+    Args:
+    conf_store: ConftStoreSearch object
+
+    Returns: list of node ids
+    """
+    data_node_ids = ConftStoreSearch.get_data_pods(_INDEX)
+    return data_node_ids
+
+
+def get_server_nodes() -> list:
+    """
+    Fetches server node ids using HA wrapper class and displays the result.
+
+    Args:
+    conf_store: ConftStoreSearch object
+
+    Returns: list of node ids
+    """
+    server_node_ids = ConftStoreSearch.get_server_pods(_INDEX)
+    return server_node_ids
+
+
+# pylint: disable=redefined-outer-name
+def get_disks(args: argparse.Namespace) -> None:
+    """
+    Fetches disk ids using ConfStore search API and displays the result.
+
+    Args:
+    conf_store: ConftStoreSearch object
+    node_id: machine_id value
+    """
+    disk_ids = ConftStoreSearch.get_disk_list(_INDEX, args.node_id)
+    print(disk_ids)
+
+
+# pylint: disable=redefined-outer-name
+def get_cvgs(args: argparse.Namespace) -> None:
+    """
+    Fetches cvg ids using ConfStore search API and displays the result.
+
+    Args:
+    conf_store: ConftStoreSearch object
+    node_id: machine_id value
+    """
+    cvg_ids = ConftStoreSearch.get_cvg_list(_INDEX, args.node_id)
+    print(cvg_ids)
+
+
+# pylint: disable=too-many-locals, redefined-outer-name, broad-except, unspecified-encoding
+# pylint: disable=protected-access, inconsistent-return-statements
+def publish(args: argparse.Namespace) -> None:
```

Comment:
We have an option of disabling pylint for this file itself.

Reply:
Earlier it was disabled at file level only but as per your [comment](https://github.com/Seagate/cortx-test/pull/1398#discussion_r839287489) i changed it to module level.